src/axom/inlet/Container.cpp

```diff
@@ -578,6 +582,24 @@ void addIndexViewToGroup(sidre::Group& group, const VariantKey& index)
   }
 }
 
+void updateUnexpectedNames(const std::string& accessed_name,
```

Comment:
Naming conventions: `accessed_name` -> `accessedName` and `unexpected_names` -> `unexpectedNames`.